README.md

```diff
@@ -19,7 +19,7 @@ npm install
 
 ### Start App with API url
 ```
-API_URL="{api_url}" npx expo start --tunnel
+API_URL="{192.168.178.121}" npx expo start --tunnel
```

Comment:
bitte wieder auf die variable ändern, die ip Adresse des backend hosts ist in jedem Netzwerk anders